patches/expo-av+13.0.1.patch

```diff
@@ -1,3 +1,51 @@
+diff --git a/node_modules/expo-av/CHANGELOG.md b/node_modules/expo-av/CHANGELOG.md
+index 269878f..9f55520 100644
+--- a/node_modules/expo-av/CHANGELOG.md
++++ b/node_modules/expo-av/CHANGELOG.md
+@@ -10,12 +10,6 @@
+ 
+ ### 💡 Others
+ 
+-## 13.0.2 — 2022-11-29
+-
+-### 🐛 Bug fixes
+-
+-- Fixed error for duplicated META-INF files when building on Android. ([#20251](https://github.com/expo/expo/pull/20251) by [@kudo](https://github.com/kudo))
+-
+ ## 13.0.1 — 2022-10-30
+ 
+ ### 🎉 New features
+diff --git a/node_modules/expo-av/android/build.gradle b/node_modules/expo-av/android/build.gradle
+index 9198d33..49edd82 100644
+--- a/node_modules/expo-av/android/build.gradle
++++ b/node_modules/expo-av/android/build.gradle
+@@ -5,7 +5,7 @@ apply plugin: 'kotlin-android'
+ apply plugin: 'maven-publish'
+ 
+ group = 'host.exp.exponent'
+-version = '13.0.2'
++version = '13.0.1'
```

Comment:
i don't know why but even clean install, running patch-package with just JS changes is changing the build.gradle file, some post install script is doing some magic. Ignoring for now